src/app/components/portfolio/portfolio.js

```diff
@@ -0,0 +1,26 @@
+var slideIndex = 1;
+showSlides(slideIndex);
+
+function plusSlides(n) {
+  showSlides(slideIndex += n);
+}
+
+function currentSlide(n) {
+  showSlides(slideIndex = n);
+}
+
+function showSlides(n) {
+  var i;
+  var slides = document.getElementsByClassName("mySlides");
+  var bars = document.getElementsByClassName("bar");
```

Comment:
jak opakujesz a Array.from to bedzie tablica i będzie można zrobić foreacha po niej